install_ffmpeg.sh

```diff
@@ -75,47 +75,88 @@ if [ ! -e "$ROOT/x264" ]; then
   make install-lib-static
 fi
 
+if [[ $(uname) == "Linux" && $BUILD_TAGS == *"debug-video"* ]]; then
+  if [ ! -e "$ROOT/x265" ]; then
+    sudo apt-get install -y libnuma-dev cmake
+    git clone https://bitbucket.org/multicoreware/x265_git.git "$ROOT/x265"
+    cd "$ROOT/x265"
+    git checkout 17839cc0dc5a389e27810944ae2128a65ac39318
+    cd build/linux/
+    cmake -DCMAKE_INSTALL_PREFIX=$ROOT/compiled -G "Unix Makefiles" ../../source
+    make
+    make install
+  fi
+  # VP8/9 support
+  if [ ! -e "$ROOT/libvpx" ]; then
+    git clone https://chromium.googlesource.com/webm/libvpx.git "$ROOT/libvpx"
+    cd "$ROOT/libvpx"
+    git checkout ab35ee100a38347433af24df05a5e1578172a2ae
+    ./configure --prefix="$ROOT/compiled" --disable-examples --disable-unit-tests --enable-vp9-highbitdepth --enable-shared --as=nasm
+    make
+    make install
+  fi
+fi
+
 EXTRA_FFMPEG_FLAGS=""
 EXTRA_LDFLAGS=""
+# all flags which should present for production build, but should be replaced/removed for debug build
+DEV_FFMPEG_FLAGS="--disable-programs"
+FFMPEG_MAKE_EXTRA_ARGS=""
 
 if [ $(uname) == "Darwin" ]; then
   EXTRA_LDFLAGS="-framework CoreFoundation -framework Security"
 else
   # If we have clang, we can compile with CUDA support!
   if which clang > /dev/null; then
     echo "clang detected, building with GPU support"
-
-    EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid --enable-filter=scale_cuda,signature_cuda --enable-encoder=h264_nvenc"
-
+    EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid,hevc_cuvid,vp8_cuvid,vp9_cuvid --enable-filter=scale_cuda,signature_cuda,hwupload_cuda --enable-encoder=h264_nvenc,hevc_nvenc"
     if [[ $BUILD_TAGS == *"experimental"* ]]; then
+        if [ ! -e "$ROOT/compiled/lib/libtensorflow_framework.so" ]; then
```

Comment:
If we're adding this here, mind removing the installation step [here](https://github.com/livepeer/go-livepeer/blob/a8a3ede471a67162bc2246c388ad93c82a109be9/docker/Dockerfile.build-linux#L25)?

Reply:
Done, also switched to libtensorflow version with both CPU and GPU support.

Reply:
Looks like we're still using libtensorflow 2.3.0. Does that have CPU & GPU support?

Reply:
Yes, GPU version supports CPU.